pallets/did/src/mock.rs

```diff
@@ -277,84 +277,44 @@ pub fn get_did_identifier_from_ecdsa_key(public_key: ecdsa::Public) -> DidIdenti
 	MultiSigner::from(public_key).into_account()
 }
 
-pub fn get_ed25519_authentication_key(default: bool) -> ed25519::Pair {
-	if default {
-		ed25519::Pair::from_seed(&DEFAULT_AUTH_SEED)
-	} else {
-		ed25519::Pair::from_seed(&ALTERNATIVE_AUTH_SEED)
-	}
+pub fn get_ed25519_authentication_key(seed: &[u8; 32]) -> ed25519::Pair {
```

Comment:
If we already start to refactor our mocks and tests, I would rather see these functions into the `mock_util`. The mock should only contain mocked pallets and the `ExtBuilder`.

Reply:
Alright. Will create some tickets.